src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace UnzerSDK\Resources\EmbeddedResources\Paypage;
+
+use UnzerSDK\Resources\AbstractUnzerResource;
+
+class PaymentMethodConfig extends AbstractUnzerResource
+{
+    protected ?bool $enabled = null;
+    protected ?int $order = null;
+
+    // type specific configs.
+    protected ?string $label = null; // paylater types only.
+    protected ?string $checkoutType = null; // paypal only.
```

Comment:
doesn't exist for paypal. Express doesn't make sense for PP

Reply:
However, a checkoutType on the create paypage call exists full/payment_only/no_shipping

Reply:
Ok, I can remove it.
that means the checkouttType for methodconfig will also be removed on paypage service?
I saw it just for paypal there.